### ISSUE TITLE
Unit testing: object in global environment not being seen in testing.

### ISSUE THREAD

**Issue by Tess-LaCoil** [traitecoevo/rmot#7]
Problem in testing for constant single and multiple individual models, y_single not seen despite being in global environment. Works just fine when run in interactive mode.

```
── Error ('test-rmot_models.R:41:5'): Execution and output: Constant single individual ──
<purrr_error_indexed/rlang_error/error/condition>
Error in `.f(.x[[i]], ...)`: i In index: 4.
i With name: time.
Caused by error in `.f()`:
! object 'y_single' not found
Backtrace:
     ▆
  1. ├─rmot::rmot_run(...) at test-rmot_models.R:41:4
  2. ├─rmot::rmot_assign_data(...) at rmot/R/rmot_run.R:18:2
  3. │ └─purrr::map(user_code, eval) at rmot/R/rmot_assign_data.R:21:2
  4. │   └─purrr:::map_("list", .x, .f, ..., .progress = .progress)
  5. │     ├─purrr:::with_indexed_errors(...)
  6. │     │ └─base::withCallingHandlers(...)
  7. │     ├─purrr:::call_with_cleanup(...)
  8. │     └─base (local) .f(.x[[i]], ...)
  9. │       └─base (local) .f(.x[[i]], ...)
 10. └─base::.handleSimpleError(...)
 11.   └─purrr (local) h(simpleError(msg, call))
 12.     └─cli::cli_abort(...)
 13.       └─rlang::abort(...)
```

**Comment by fontikar** [comment]
Relevant:

 https://github.com/r-lib/testthat/issues/1486

**Comment by fontikar** [comment]
Fixed here 797316cd9c5a2e02a74420aa75f6618e52fca0eb 
 I was not using quosures for rmot_assign_data which mean the expressions typed by user were hanging out in a limbo environment. Changing rlang::enexprs to rlang::enquos did the trick
